editor/editor_node.cpp

```diff
@@ -317,35 +317,40 @@ void EditorNode::shortcut_input(const Ref<InputEvent> &p_event) {
 
 	Ref<InputEventKey> k = p_event;
 	if ((k.is_valid() && k->is_pressed() && !k->is_echo()) || Object::cast_to<InputEventShortcut>(*p_event)) {
-		EditorPlugin *old_editor = editor_plugin_screen;
-
+		bool is_handled = false;
```

Comment:
How about defaulting to true and only setting it to false in a final `else`?
Should save a bunch of repetition.

Reply:
Lol! So simple, really better! That's brilliant!! :)
Done!